odoo/addons/base/models/ir_actions.py

```diff
@@ -684,59 +684,41 @@ def _compute_crud_relations(self):
-                if isinstance(field, fields.Json):
-                    raise ValidationError(_("I'm sorry to say that JSON fields (such as %s) are currently not supported.", field_name))
-        target_records = None
-        if record is not None:
-            target_records = reduce(getitem, path[:-1], record)
-        model_id = self.env['ir.model']._get(Model._name)
-        field_id = self.env['ir.model.fields']._get(Model._name, field_name)
-        return model_id, field_id, target_records
-
-    def _stringify_path(self):
-        """ Returns a string representation of the update_path, with the field names
-        separated by the `>` symbol."""
+        field_chain, _field_chain_str = self._get_relation_chain(field_name)
+        last_field = field_chain[-1]
+        if isinstance(last_field, fields.Json):
+            raise ValidationError(_("I'm sorry to say that JSON fields (such as '%s') are currently not supported.", last_field.string))
+        model_id = self.env['ir.model']._get(last_field.model_name)
+        field_id = self.env['ir.model.fields']._get(last_field.model_name, last_field.name)
+        return model_id, field_id
+
+    def _get_relation_chain(self, searched_field_name):
         self.ensure_one()
-        path = self.update_path
+        path = self[searched_field_name].split('.')
         if not path:
-            return ''
+            return []
         model = self.env[self.model_id.model]
-        pretty_path = []
-        field = None
-        for field_name in path.split('.'):
-            if field and field.type == 'properties':
-                pretty_path.append(field_name)
-                continue
+        chain = []
+        # field = None
+        for field_name in path:
+            is_last_field = field_name == path[-1]
             field = model._fields[field_name]
-            field_id = self.env['ir.model.fields']._get(model._name, field_name)
-            if field.relational:
+            if not is_last_field:
+                if not field.relational:
+                    # sanity check: this should be the last field in the path
+                    searched_field = self._fields[searched_field_name]
+                    raise ValidationError(_("The path contained by the field '%(searched_field)s' contains a non-relational field (%(current_field)s) that is not the last field in the path. You can't traverse non-relational fields (even in the quantum realm). Make sure only the last field in the path is non-relational.", searched_field=searched_field.string, current_field=field_name))
                 model = self.env[field.comodel_name]
-            pretty_path.append(field_id.field_description)
-        return ' > '.join(pretty_path)
+            chain.append(field)
+        stringified_path = ' > '.join([field.string for field in chain])
```

Comment:
I suppose you should use field.get_descrition(env)["name"] instead to have a translated value for the field's string